Source/Krypton Components/Krypton.Navigator/Palette/PaletteRedirectRibbonTabContent.cs

```diff
@@ -160,24 +160,24 @@ public PaletteRedirectRibbonTabContent(PaletteBase target)
         /// </summary>
         public virtual void ResetRedirectStates()
         {
-            _disabledBack = null;
-            _normalBack = null;
-            _pressedBack = null;
-            _trackingBack = null;
-            _selectedBack = null;
-            _focusOverrideBack = null;
-            _disabledText = null;
-            _normalText = null;
-            _pressedText = null;
-            _trackingText = null;
-            _selectedText = null;
-            _focusOverrideText = null;
-            _disabledContent = null;
-            _normalContent = null;
-            _pressedContent = null;
-            _trackingContent = null;
-            _selectedContent = null;
-            _focusOverrideContent = null;
+            _disabledBack = null!;
```

Comment:
This is clearly a null assignment, so make the fields nullable, and fix the fallout.